easybuild/scripts/bootstrap_eb.py

```diff
@@ -603,90 +618,96 @@ def main():
 #         default=DEFAULT_URL,
 #         help='alternative URL from where to download the distribute package')
 DISTRIBUTE_SETUP_PY = """
```

Comment:
changing this but without any changes to the comments above?

is there any feasible way we can avoid doing this btw?

since we need to download stuff anyway, what's the use of including this as a blob?